ufm-test-services/metamask/Dockerfile

```
@@ -1,10 +1,16 @@
-FROM node:18
+FROM mcr.microsoft.com/playwright:v1.37.1-jammy
 
 WORKDIR /app
 
-COPY ./ ./
+ENV PATH /app/node_modules/.bin:$PATH
 
-RUN npm i -g pnpm
-RUN pnpm install
+COPY tests /app/tests/
+COPY .env /app/
+COPY package.json /app/
+COPY playwright.config.ts /app/
+COPY start.sh /app/
+COPY tsconfig.json /app/
+
+RUN npm install
```

```suggestion
COPY package.json /app/

RUN npm install

COPY tests /app/tests/
COPY .env /app/
COPY playwright.config.ts /app/
COPY start.sh /app/
COPY tsconfig.json /app/
```
In docker you want to copy the minimum files necessary before running a command. This way your "cache hits" are higher. By doing it this way when you change anything that is not package.json it won't have to redownload node modules it will just have them cached and build faster